### ISSUE TITLE
v1 js function debug

### ISSUE THREAD

**Issue by JohnReeson** [Microsoft/vscode-azurefunctions#770]
Create a v1 js function project, add a http trigger. F5.

No debugger attached and only get message in Debug console

    Debugging with inspector protocol because Node.js version could not be determined (Error: connect ECONNREFUSED 127.0.0.1:5858)

Try [task.json for v2](https://github.com/Microsoft/vscode-azurefunctions/blob/master/docs/debugConfig.md). Got 

    Cannot connect to runtime process, timeout after 10000 ms - (reason: Cannot connect to the target: connect ECONNREFUSED 127.0.0.1:5858)

Close this error info and F5 again, I can debug now. Any idea?

**Comment by EricJizbaMSFT** [comment]
What version of node do you have installed on your machine? Can you try adding `"protocol": "legacy"` to your `.vscode\launch.json` file? It will probably look like this after you add that:
```json
{
  "version": "0.2.0",
  "configurations": [
    {
      "name": "Attach to JavaScript Functions",
      "type": "node",
      "request": "attach",
      "protocol": "legacy",
      "port": 5858,
      "preLaunchTask": "runFunctionsHost"
    }
  ]
}

```

**Comment by JohnReeson** [comment]
I manage to debug with two steps:

1. In launch.json, add `"protocol": "legacy"` and delete `"preLaunchTask": "runFunctionsHost"`.
2. In terminal ,`func host start` to manually start host and F5.

**Comment by EricJizbaMSFT** [comment]
@JohnReeson according to [this doc](https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions), v1 officially works on Node 6. If I spent time on this bug, I would test on Node 6 and not Node 8 or 10.

We're also admittedly focused on v2 and not v1, for a variety of reasons (because it's cross platform like VS Code and because it's the latest version). Do you have any plans to move over to v2 any time soon?

I'm not saying I won't look into this bug - but just letting you know it's lower priority (hopefully you unblocked yourself anyways)

**Comment by JohnReeson** [comment]
@EricJizbaMSFT  Sorry about the mistake and thanks for your patience. In my test with node 6, the same thing happens, but I will just move to v2 right now. Thanks again, not a real block so I just close it.
